backend/src/main/java/de/tomze/backend/service/UserService.java

```diff
@@ -51,16 +61,13 @@ public UserEntity updateUser(String userName, UserFromAppDto userFromAppDto) {
             throw new IllegalArgumentException("NotFound");
         }
         UserEntity userEntityToUpdate = userEntityOptional.get();
-        if (userEntityToUpdate.equals(userFromAppDto)) {
-            throw new IllegalArgumentException("Nothing to Change");
-        }
-        deleteUser(userEntityToUpdate.getUserName());
+        deleteUser(userEntityToUpdate);
```

Comment:
sobald deine User eine Historie bekommen (gespeicherte Daten oder ähnliches) würden Sie dann ja auch verlieren bei einem update des Profiles. Nur als Idee für den Hinterkopf, dass das in der Zukunft für Bugs sorgen könnte